cmis/exceptions.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
```

Comment:
@sbidoul I wonder if it would be better to delete these exceptions and to use only those available in Odoo